suse-open-source-policy.html

```diff
@@ -93,29 +93,29 @@ <h2 id="introduction">Introduction</h2>
+    <p>When we discuss Open Source Software we refer to the definition of <a href="https://opensource.org/licenses">Open Source Licenses by the Open Source Initiative ("OSI")</a>. SUSE considers software, including its documentation, released under a license that is compatible with OSI's Open Source Definition to be Open Source.</p>
 
     <h2 id="producing-open-source-software">Producing Open Source Software</h2>
 
-    <p>We develop in the open and publish our code as Open Source. SUSE products are fully Open Source. There are only rare exceptions where we don’t release code as Open Source. Our development model is designed around Open Source to make it easy for our engineers to produce and contribute to Open Source software.</p>
+    <p>We develop in the open and publish our code as Open Source. SUSE products are fully Open Source. There are only rare exceptions where we don't release code as Open Source. Our development model is designed around Open Source to make it easy for our engineers to produce and contribute to Open Source software.</p>
 
     <h3 id="contributing-to-open-source-projects">Contributing to Open Source Projects</h3>
 
-    <p>SUSE considers itself as part of the global Open Source community. When we fix bugs or make other changes we operate under the maxime “upstream first”. That means we contribute all changes back to the community as early as possible so that they become part of the common base we build on.</p>
+    <p>SUSE considers itself as part of the global Open Source community. When we fix bugs or make other changes we operate under the maxime "upstream first". That means we contribute all changes back to the community as early as possible so that they become part of the common base we build on.</p>
 
     <h4 id="contributing-patches">Contributing Patches</h4>
 
     <p>When contributing to Open Source projects, follow the guidelines of those upstream projects. Respect their governance model and contribution policies. Use the license of the upstream project for your contribution provided it is a license that is compatible with the Open Source Definition.</p>
 
     <h4 id="contributing-non-code">Contributing Non-Code</h4>
 
-    <p>There are many ways to contribute to Open Source projects other than writing code: Writing documentation, reporting bugs, helping users, discussing development, etc. Apply the spirit of the guideline here as well. Be open, respectful, and collaborate.</p>
+    <p>There are many ways to contribute to Open Source projects other than writing code: Writing documentation, reporting bugs, helping users, discussing development, etc. Apply the spirit of the guideline here as well. Be open, respectful, and collaborate. Use the agreed/accepted license of the upstream project also for non-code contributions provided it is a license that is compatible with the Open Source Definition.</p>
```

Comment:
I'm not sure why that would be restricted to non-code submissions only. SUSE generally  would like to use the commonly accepted upstream license for any new additions to the upstream projected assuming if it is an OSI compatible license. if it isn't we're picking one that is.

Reply:
If you look at the list of approved licenses at license.opensuse.org we have a large list of exceptions for OSI compatible licenses. We also have a well defined process for legal to review these exceptions which I imagine would be much harder for non code submissions etc.

Reply:
I am not sure we're talking about the same thing. I think you're referring to the SUSE-XXX license declarations that we had to add to "invent" license declarations that are not handled by spdx.org already, which we use for license declarations in general. The SUSE- prefix doesn't mean "SUSE invented the license" it means "SUSE had to invent a handle for this nonstandardized license of this project we consume". 

the quoted paragraph is about which license a SUSE employee should contribute to upstream projects as non-code. and that is restricted by "employees need to contribute under OSI approved licenses", because everything else makes legal's IP handling difficult.